_includes/sidebar.html

```diff
@@ -12,15 +12,22 @@
     <nav class="sidebar-nav">
       {% for sections in site.contents[page.lang] %}
         {% assign _section = sections[0] | strip %}
+        {% if page.section == _section %}
+          {% assign opened = true %}
+        {% elsif page.section == "home" and forloop.index0 == 1 %}
+          {% assign opened = true %}
```

Comment:
@eksperimental, great idea. But I've a question: should we do something with the values of `sections.home` in the [locales](https://github.com/doomspork/elixir-school/pull/1112/files#diff-55286c7d0270bebf2261ad00b1964c8dR8)? The link to the index will be called 'Home' for every language if we leave those in the current state.